### DISCUSSION TITLE
Unable to register new account?

### DISCUSSION THREAD

**Discussion by TheSkallywag** [civitai/civitai#218] [Q&A]
I have the software running successfully on my localhost:3000 (npm run dev). I try to register a new account, then after I get the email with a special link that brings me to this page and it says login error.

http://localhost:3000/login?error=Verification

What am I doing wrong?

**Comment by manuelurenah** [comment]
Hey @TheSkallywag! If I understood correctly, you have cloned our project and are trying to run it locally in your computer, but getting an error when you try to login using one of the social providers.

For it to be successful, you'll need to setup the env vars in a `.env` file to let the app and server know which db to connect to, specify your API tokens for allowing logins using the social providers and whatnot. We cannot provide you with those values atm, so you'll have to generate those yourself. If you're wanting to run the app to actually help us with development, please reach out to any of the dev team members in our discord server 🙂 https://discord.gg/AschR3fF
